sycl/test-e2e/Plugin/interop-opencl-make-kernel.cpp

```diff
@@ -45,6 +45,9 @@ int main() {
   assert(OpenCLError == CL_SUCCESS);
 
   kernel Kernel = make_kernel<backend::opencl>(OpenCLKernel, Context);
+  // Should not throw an exception
```

Comment:
I suggest to wrap the call into `try..catch` and an assert into `catch` block. This way if we break the exception logic, it would be more clear what exactly went wrong in the logs